### PR TITLE
Add WSL2 support

### PR DESCRIPTION
**Pull request by ryanvoecks**
- Update Windows installation README to use WSL2
- Add installation for Windows usbipd to WSL setup script
- Add script `attach_usb.sh` for connecting Vbuddy to WSL
- Update README to clarify that native Linux is not supported
- Other small clarity improvements to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to Lab 0 of Autumn Term [Instruction Architectures and Compilers](http://intranet.ee.ic.ac.uk/electricalengineering/eecourses_t4/course_content.asp?c=ELEC50010&s=I2) (IAC).
 
-In this lab, you will install the tools needed for both the labs and coursework.
+In this lab, you will install the tools needed for both the labs and coursework. The labs for this term can be run on both macOS and Windows. Native Linux is not yet supported.
```
Comment (saturn691) [inline README.md:5]: Native Linux should work? I don't see why not

Comment (ryanvoecks) [reply README.md]: Communication between Linux PC and Vbuddy is broken. This causes a hang when vbdOpen() is called.

**Comment by Jpnock** [comment]
Nice work here :)

When I looked into this a year ago, I decided it wasn't a viable approach since `usbipd` kept causing my machine to blue-screen. They had a bunch of similar reports in their GitHub Issues from what I remember too. 

I've just tested your scripts and they seem to work fine. So hopefully those driver BSOD stability problems have now been resolved.

Hopefully this also solves some of the weird communication problems that WSL1 caused when talking to vBuddy!

FYI: I pushed a commit to make `attach_usb.sh` executable since it didn't have `+x` when I ran the install script.

**Comment by ryanvoecks** [comment]
Ah, I didn't even realise that there were BSOD issues with usbipd. Haven't encountered anything like that thankfully. Thanks for pushing the patch to make `attach_usb.sh` executable!

**Pull request merged**